encrypted-media/polyfill/cast-polyfill.js

```diff
@@ -41,7 +41,7 @@
 
         function MediaKeySystemAccess( access )
         {
-            this._access = mediaKeySystemAccess;
+            this._access = access;
```

Comment:
Yes please.

On Sep 22, 2016, at 8:27 PM, Sukhmal Kommidi <notifications@github.com>
wrote:

In encrypted-media/polyfill/cast-polyfill.js
<https://github.com/w3c/web-platform-tests/pull/3795#discussion_r80116780>:

> @@ -41,7 +41,7 @@
>
>          function MediaKeySystemAccess( access )
>          {
> -            this._access = mediaKeySystemAccess;
> +            this._access = access;

@mwatson2 <https://github.com/mwatson2> Do you want me to undo the changes
to this polyfill?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/w3c/web-platform-tests/pull/3795/files/fd5d1f377942c1e04e709d0d9c9099d1917674be#r80116780>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AHPfi0S5nR7D_gofpKp11eULdcD1C8XPks5qsta6gaJpZM4KDcqf>
.